pybamm/models/full_battery_models/base_battery_model.py

```diff
@@ -400,6 +401,18 @@ def options(self, extra_options):
                 "Unknown lithium plating model '{}'".format(options["lithium plating"])
             )
 
+        if options["lithium plating porosity change"] not in ["true", "false"]:
+            if options["lithium plating porosity change"] in [True, False]:
+                raise pybamm.OptionError(
+                    "lithium plating porosity change must now be given in string format"
```

Comment:
there should be a space in the string at the end (otherwise it will say "format(true")